docker/Dockerfile.test

```diff
@@ -1,9 +1,32 @@
-FROM screenly/srly-ose-base:latest-x86
+FROM screenly/srly-ose-base:latest-x86 as builder
+
+WORKDIR /opt
+RUN cd /opt
+RUN git clone https://git.ffmpeg.org/ffmpeg.git
+WORKDIR ffmpeg
+RUN git checkout 2ca65fc7b74444edd51d5803a2c1e05a801a6023 && \
+    ./configure --disable-x86asm && make
+
+FROM builder
```

Comment:
This is where it goes wrong. You need to use `FROM screenly/srly-ose-base:latest-x86` here. Consider everything *above* this throwaway. You just want to *copy* data from it. You don't want to actually use this image at al.l.